pom.xml

```diff
@@ -3,7 +3,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.exasol</groupId>
     <artifactId>virtual-schema-common-java</artifactId>
-    <version>5.2.1</version>
+    <version>5.3.1</version>
```

Comment:
I'm not sure if I should change the major version? It makes adapters implement the new method

Reply:
Changing the minor version is correct. While it adds a new function, I leaves the existing interface intact. No breaking change --> no change in major number.